component/Goals/GoalsList.jsx

```diff
@@ -181,8 +191,7 @@ function GoalsList({ user, isArchived = false }) {
       })
       .then(({ data: response }) => {
         if (response.status === 200) {
```

Comment:
comment mentioned above. What other status other than 200 is success

Reply:
this is not a network status it is the custom status that we passing in the response